Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -53,7 +53,20 @@ const KeyboardAvoidingView = React.createClass({
   propTypes: {
     ...View.propTypes,
     behavior: PropTypes.oneOf(['height', 'position', 'padding']),
+    
+    /**
+     * The style of the content container(View) when behavior is 'position'.
+     */
+    contentContainerStyle (props, propName, componentName) {
+      const isStyle = View.propTypes.style;
 
+      if (props[propName] && props.behavior !== 'position') {
```

Comment:
I wouldn't bother with this, it doesn't work with Stylesheets or with an array of styles. Just use View.propTypes.style for the whole thing.